Documentation/ApiOverview/Seo/ExampleWebsitetitle/_WebsiteTitleProvider.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+declare(strict_types=1);
+
+namespace MyVendor\MySitepackge\PageTitle;
+
+use TYPO3\CMS\Core\PageTitle\AbstractPageTitleProvider;
+use TYPO3\CMS\Core\Site\SiteFinder;
+
+final class WebsiteTitleProvider extends AbstractPageTitleProvider
```

Comment:
Hmm, I don't like this implementation as it uses $GLOBALS['TSFE']. This should not be used in a best practise. Someday it will be gone.

Also, when I set the site configuration option "Website title" the title gets prefixed automatically (okay, not postfixed).

Reply:
I actually took this part from what the Core is doing:

https://github.com/TYPO3/typo3/blob/main/typo3/sysext/core/Classes/PageTitle/RecordPageTitleProvider.php#L27

https://github.com/TYPO3/typo3/blob/main/typo3/sysext/seo/Classes/PageTitle/SeoTitlePageTitleProvider.php#L30

What would you suggest doing instead?

Reply:
Oh my :-(
Thought about the request object, but page title isn't there.
Can we move then the `$GLOBALS['TSFE']` into a `getTypoScriptFrontendController` method? This way it looks perhaps a little bit better.

I suggest to implement the interface instead extending the abstract class. The constructor gets the SiteFinder object, and the `getTitle()` method does the work assembling the title. Looks a bit cleaner to me. What do you think?

Reply:
@linawolf any news, any answer to Chris?

Reply:
@linawolf @brotkrueml can this be dismissed and the issue with the latest commit be merged?